src/scripts/content/khanacademy.js

```diff
@@ -2,16 +2,28 @@
 /*global $: false, document: false, togglbutton: false*/
 'use strict';
 
-togglbutton.render('#task:not(.toggl)', {observe: true}, function (elem) {
-  var link,
-    description = $('.title', elem).textContent,
-    project = $('.project', elem).textContent;
+togglbutton.render('#tutorial-content [data-reactid="5"]:not(.toggl)', {}, function (elem) {
```

Comment:
I'm not really fond of using this `data-reactid` on the selector as I think this might change someday.
But they're using custom class names, which would be even less reliable.